pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -689,17 +707,12 @@ impl RemoteTimelineClient {
             }
 
             // schedule the actual deletions
-            for (name, generation) in with_generations {
-                let op = UploadOp::Delete(Delete {
-                    file_kind: RemoteOpFileKind::Layer,
-                    layer_file_name: name.clone(),
-                    scheduled_from_timeline_delete: false,
-                    generation,
-                });
-                self.calls_unfinished_metric_begin(&op);
-                upload_queue.queued_operations.push_back(op);
-                info!("scheduled layer file deletion {name}");
```

Comment:
The reason why we've been logging the names is that we can afterwards understand the lifecycle of a layer. It has been quite useful in a number of ... at least eviction related situations.

Reply:
Reinstated a log line per layer in 28d7f6b64